dataflow/utils.py

```diff
@@ -99,6 +99,9 @@ class TableConfig:
     # There are few good reasons not to prefer using this option that applies the indexes after ingestion.
     indexes: Optional[List[LateIndex]] = None
 
+    # Constraints to apply on the top-level table
```

Comment:
The comment about "top-level" table is maybe slightly confusing - I think just drop that phrase? It could make someone think that nested tables can't have constraints?

```suggestion
    # Constraints to apply on the table
```

Reply:
Sounds good